pmlc/dialect/tile/contraction.h

```diff
@@ -48,7 +48,7 @@ struct Constraints {
 };
 
 struct Contraction {
-  explicit Contraction(ContractionOp op);
+  explicit Contraction(ContractionOp op, llvm::ArrayRef<ConstraintOp> constraintOps);
```

Comment:
We usually only use `explicit` with single-parameter constructors -- are you trying to disallow copy-list initialization, or is this left over from previously?

Reply:
Oops just leftover, thanks for catching this.